### ISSUE TITLE
Deprecation warning due to invalid escape sequences

### ISSUE THREAD

**Issue by tirkarthi** [vovanec/supervisor_checks#13]
Deprecation warnings are raised due to invalid escape sequences. This can be fixed by using raw strings or escaping the literals. pyupgrade also helps in automatic conversion : https://github.com/asottile/pyupgrade/

```
find . -iname '*.py' | grep -Ev 'test.py' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}
./supervisor_checks/bin/complex_check.py:3: DeprecationWarning: invalid escape sequence \/
  """Example configuration:
```
